app-crypt/rhash/rhash-1.3.5.recipe

```diff
@@ -70,3 +70,10 @@ INSTALL()
 	packageEntries devel \
 		$developDir
 }
+
+TEST()
+{
+	cd tests
+	LIBRARY_PATH="$(pwd)/../librhash${LIBRARY_PATH:+:$LIBRARY_PATH}" \
```

Comment:
You might wish to replace `$(pwd)/../` by `$sourceDir/`
That would be equivalent, since the working directory where `TEST()` starts is $sourceDir (which is the root directory of the tarball).

BTW, like @pulkomandy wrote, you did a really great work!